github-profiles/Johan-Jimenez-4558/README.md

```diff
@@ -0,0 +1,40 @@
+# Hola! 👋, soy Johan Sebastian Jimenez Molina
+
+>[Mi perfil de Github](https://github.com/jsebas2220)
+
+•	Estudiante de ingeniería en sistemas, especialista en soluciones de soporte técnico de software y hardware, actualmente en formación desarrollo web Full Stack, con capacidad para la gestión y diseño y testing de proyectos de software, conocimientos en lenguajes de programación (HTML, Css, Bootstrap, JavaScript, Java), operación de bases de datos ( Oracle, MySql, SQL, MariaDB) conocimientos en manejo de comandos  Linux con distribuciones RedHat y CentOS, Cloud AWS, , conocimientos de metodologías agiles para el desarrollo de software y metodologías Itil V4 para la operación de servicio
+
+
+---
+
+## 🔧 Stack de Tecnologías
+
+| Lenguajes de Programación | Frontend | Herramientas       |
+| ------------------------- | -------- | ------------------ |
+| ![JavaScript](https://img.shields.io/badge/javascript-%23323330.svg?style=for-the-badge&logo=javascript&logoColor=%23F7DF1E) | ![HTML5](https://img.shields.io/badge/html5-%23E34F26.svg?style=for-the-badge&logo=html5&logoColor=white)  ![CSS3](https://img.shields.io/badge/css3-%231572B6.svg?style=for-the-badge&logo=css3&logoColor=white) | ![Git](https://img.shields.io/badge/git-%23F05033.svg?style=for-the-badge&logo=git&logoColor=white)  ![GitHub](https://img.shields.io/badge/github-%23121011.svg?style=for-the-badge&logo=github&logoColor=white)  ![Visual Studio Code](https://img.shields.io/badge/Visual%20Studio%20Code-0078d7.svg?style=for-the-badge&logo=visual-studio-code&logoColor=white) |
+
+---
+
+## 🖥️ Comandos de Terminal
+
+| Comando | Descripción |
```

Comment:
Podrías alinear las tablas para hacer mas legible el markdown puro.

Reply:
Perfecto to corregido muchas gracias